phplintinstall

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Download PHPCS
+mkdir bin
+cd bin
+curl -OL https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar
+
+# Clone Drupal Coder repo
+mkdir drupal
+cd drupal
+git clone https://git.drupalcode.org/project/coder.git
```

Comment:
Since we don't really care about the repository history here, we can do a shallow clone which will make git clone only the latest revision. Also, you can leave it up to git to create the folder for you. Finally, to avoid errors when running the script multiple times, it's important to first check if the destination folder exists:

```
if [ ! -d drupal/coder ]; then
  git clone --depth 1 https://git.drupalcode.org/project/coder.git drupal/coder
fi
```